phylogenetic/rules/prepare_sequences.smk

```diff
@@ -3,48 +3,41 @@ This part of the workflow prepares sequences for constructing the
 phylogenetic tree.
 """
 
-# FIXME once the ingest build is finalized and data is uploaded to S3,
-# this rule should be uncommented, and the location of metadata.tvs
-# and sequences.fasta in various rules should be adjusted.
-# rule download:
-#     output:
-#         metadata="data/metadata.tsv.zst",
-#         sequences="data/sequences.fasta.zst",
-#     params:
-#         sequences_url="https://data.nextstrain.org/files/workflows/yellow-fever/sequences.fasta.zst",
-#         metadata_url="https://data.nextstrain.org/files/workflows/yellow-fever/metadata.tsv.zst",
-#     shell:
-#         r"""
-#         curl -fsSL --compressed {params.sequences_url:q} --output {output.sequences}
-#         curl -fsSL --compressed {params.metadata_url:q} --output {output.metadata}
-#         """
+
+rule download:
+    output:
+        metadata="data/metadata.tsv.zst",
+        sequences="data/sequences.fasta.zst",
+    params:
+        sequences_url="https://data.nextstrain.org/files/workflows/rubella/sequences.fasta.zst",
+        metadata_url="https://data.nextstrain.org/files/workflows/rubella/metadata.tsv.zst",
+    shell:
+        r"""
+        curl -fsSL --compressed {params.sequences_url:q} --output {output.sequences}
+        curl -fsSL --compressed {params.metadata_url:q} --output {output.metadata}
+        """
```

Comment:
[beyond this PR]

Evey time I review a new pathogen build I want to run ingest and phylo locally, and I have to either modify the snakefile or `cp` the ingest outputs into `phylogenetic/data`. It'd be great to take the repo template design in a direction that natively supports using local ingest files. (I've done this for `avian-flu` with my `{additional_,}inputs` interface.)

Reply:
A similar comment about external analysis directories - I'm not saying we should be implementing them ad-hoc in repos, but if they became part of the template then we'd have them here. For Rubella specifically I ended up making a few changes to the workflow (rooting etc), but wanted to keep each build so I could compare them. That's a lot of manual `mv X Y` commands rather than separate analysis directories which preserve the `config` values etc.

Reply:
Just to provide explanation/context on my decision making here: I'm kinda pausing on #13 until there's a more clear signal about what our preferred implementation pattern looks (i.e., until the pathogen repo guide and zika builds get updated with whatever the pattern is). 

Agree with you about the local ingest option; the pattern I've used for the last couple pathogen builds I've done is to initially have the phylogenetic workflow reading inputs from `../ingest/results`, and then only upload to S3 and switch over to the download-and-extract method once I'm sure the phylo build is mostly done.